src/app/components/layout/MultiSelector.js

```diff
@@ -190,4 +205,25 @@ class MultiSelector extends React.Component {
   }
 }
 
+MultiSelector.defaultProps = {
+  allowSearch: false,
+  allowToggleAll: false,
+  cancelLabel: null,
+  submitLabel: null,
+};
+
+MultiSelector.propTypes = {
+  allowSearch: PropTypes.bool,
+  allowToggleAll: PropTypes.bool,
+  cancelLabel: PropTypes.node,
+  options: PropTypes.arrayOf(PropTypes.shape({
+    value: PropTypes.string.isRequired,
+    label: PropTypes.string.isRequired,
+  })).isRequired,
+  selected: PropTypes.arrayOf(PropTypes.string).isRequired,
+  submitLabel: PropTypes.node,
+  onDismiss: PropTypes.func.isRequired,
+  onSubmit: PropTypes.func.isRequired,
+};
+
 export default MultiSelector;
```

Comment:
@amoedoamorim did you test these changes in all places where `<MultiSelector />` is used?

Reply:
Yes, and :LGTM: !